host-interaction/mutex/create-semaphore.yml

```
@@ -0,0 +1,12 @@
+rule:
+  meta:
+    name: create semaphore
+    namespace: host-interaction/mutex
+    author: alyssa.rahman@fireeye.com
+    scope: file
+    examples:
+      - 3db3e55b16a7b1b1afb970d5e77c5d98
+  features:
+    - and:
+      - os: linux
+      - import: sem_init
```

```suggestion
      - api: sem_init
```

and change scope to bb or function

---

same comment on other semaphore rules